.env

```diff
@@ -46,6 +46,7 @@ MONGO_INITDB_ROOT_USERNAME=root
 MONGO_INITDB_ROOT_PASSWORD=test
 ROBOTOFF_URL=http://robotoff.openfoodfacts.localhost:5500 # connect to Robotoff running in separate docker-compose deployment
 EVENTS_URL=
+FACET_URL = https://facets-kp.openfoodfacts.net/render-to-html?
```

Comment:
I would not include the "?", add it in the code.

(more intuitive for who changes this variable)

Reply:
also, for consistency, FACETS_KP_URL maybe?